web-ng/src/app/components/card-view-project/card-view-project.component.css

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.project-card {
+  background-image: url(/assets/img/happy-land.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  max-width: 400px;
+  padding: 32px;
+}
+
+.logo {
+  width: 64px;
+  height: 64px;
+}
+
+.title-project {
+  width: 100px;
+  color: #fff;
```

Comment:
Please use mat-color instead of color literals here and throughout!

Reply:
updated https://github.com/google/ground-platform/pull/643/commits/51031845e9688e9f5cb359d711887254823d2a17